Backpack/src/androidTest/java/net/skyscanner/backpack/calendar/presenter/BpkCalendarControllerTest.kt

```diff
@@ -29,15 +30,6 @@ class BpkCalendarControllerTest {
     subject = BpkCalendarControllerTestImpl()
   }
 
-  @Test
```

Comment:
Why is this removed if the default date is still there?

Reply:
There is no default date anymore just a data class 
```
CalendarDay(
  val year: Int,
  val month: Int,
  val day: Int
)
```

Reply:
But isn't this testing the default start and end date in the controller? Afaik those are still there. (DEFAULT_START_DATE and DEFAULT_END_DATE)

Reply:
Yes you are right. We are testing if we set a constant to a value then the value = constant. What value does this test bring?

Reply:
Well it's testing if the default start and end dates are what we expect. For one that test ensures that if for any reason the code is changed in a way that affects the default dates it will fail and prevent that error.

Plus it tests the default dates are start = today and end = today + 1 year

Reply:
👍 If you find it useful, I put it back.